app.js

```diff
@@ -89,6 +89,19 @@ app.get('/api/leaderboard/:name', async (req, res, next) => {
     }
 });
 
+app.get('/api/skins/available', async (req, res) => {
+    let availableSkins = {}
+
+    let files = await fs.readdir(`${process.cwd()}/public/skins`)
+
+    files.forEach(file => {
+        if (file.endsWith('.png') && file.startsWith('cone_'))
+            availableSkins[file.substring(5, file.length -4)] = `/skins/${file}`;
```

Comment:
Wait, doesn't it make more sense to define all skins in some json or yaml file when we define [their odds](https://github.com/SyberiaK/coneflip-overlay/blob/abb22a8a1dde26453018fc332a28263376509ad2/app.js#L16) manually anyway?

Reply:
never noticed this existed lol, well, some fallback option should be added as a default, when the skin is not defined here, but, at least, that's one less place where you should manually write in stuff.